sdk/javascript/vectors/catbuffer.js

```diff
@@ -10,6 +13,42 @@ import path from 'path';
 
 const JSONBigInt = JSONBigIntLib({ alwaysParseAsBig: true, useNativeBigInt: true });
 
+/**
+ * Factory for creating Symbol receipts.
+ */
+export default class ReceiptFactory {
```

Comment:
should this be in src?

Reply:
as discussed, moved both _somewhere_ down, 
in js: down to tests:
in py: down to testvectors, mostly cause imo import look _slightly_ better this way (need to import in both generator AND actual 'runner'):
```py
from testvectors.BlockFactory import BlockFactory as SymbolBlockFactory
from testvectors.ReceiptFactory import ReceiptFactory
```